src/styles/index.scss

```diff
@@ -16,28 +16,36 @@ color: #b7cbd9;
 	text-align: center;
 	color: #58a6ff;
 	margin: 8px;
+	display: none;
 }
 
-.main-card{
+.main-card {
 	padding: 20px;
 	border-radius: 5px;
 	background: #100e17;
 	box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.5);
 	margin: 7px;
-
 }
-.search-part{
+.search-part {
 	margin-top: 15px;
 	border: solid 1px rgb(39, 39, 39);
 	border-radius: 5px;
 	width: 100%;
-	display: flex;
+	// display: none;
 	justify-content: center;
+}
 
+// custom select
+
+.custom-control-label::before {
+	background-color:#d97b16;
+	border: #b7cbd9 solid 1px;
 }
 
-.search-part form{
```

Comment:
Isn't `.search-part` on the form? This requires it to be a child of `.search-part`.